Utils/ExtractVersionFromTilsesetJSON/readme.md

```diff
@@ -10,16 +10,22 @@ This script should :
  * A building is in its source state until the transaction is over then it is in its destination state. 
  *  Expects a temporal tileset existing from 2009 to 2015
  *  Creates three versions: one in 2009, one in 2012 and one in 2015 and two versions transitions between these versions
-  
-
 
 ## Requirement :  
 See the requirement inside the file "requirement.txt"  
 You can install them with the command `pip install -r requirement.txt` 
 
-## Execution :  
+## Execution with python:  
 To execute the script, follow the requirement and then execute  "compute_versiondata_from_py3dfiles.py"
 
+## Execution with docker:  
+in this folder : 
+```
+$docker build -t compute_versionData .
```

Comment:
When executing this I get `invalid argument "compute_versionData" for "-t, --tag" flag: invalid reference format: repository name must be lowercase` and the docker is not built. Changing the tag to `compute_version_data` does the trick. Can you update the doc please ?

Reply:
Changed